server/models/schema.sql

```diff
@@ -1276,13 +1276,14 @@ DROP TABLE IF EXISTS `patient_visit`;
 CREATE TABLE `patient_visit` (
   `uuid` BINARY(16) NOT NULL,
   `patient_uuid` BINARY(16) NOT NULL,
-  `date` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
-  `registered_by` smallint(5) unsigned NOT NULL,
+  `start_date` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `end_date` datetime DEFAULT NULL,
```

Comment:
`end_date` isn't used by this very simple first step, in this pull request we are only concerned with patients arriving at the hospital and being checked in. 

In the future we can either design a checkout system or allow the server to 'expire' a patient visit setting the end_date. This depends a lot on what reports we need to write and what standards we want to meet, it should be designed in an issue as an extentsion of this feature!